src/sources/docker.rs

```diff
@@ -396,7 +402,8 @@ impl DockerSource {
                                         .remove(&id)
                                         .expect("Every started ContainerId has it's ContainerState");
                                     if state.is_running() {
-                                        self.containers.insert(id.clone(), self.esb.start(id, Some(Duration::from_secs(5))));
+                                        let backoff = Some(self.backoff_duration.clone());
```

Comment:
`.clone()` should not be required, std::time::Duration impl Copy

Reply:
@fanatid Thanks! I didn't notice that `make check-clippy` fails. Is anyone on the action to move `EmbarkStudios/cargo-deny-action` somewhere else? It always fails at the moment and is distracting other checks.

Reply:
It's strange that clippy did not catch this. I also receive a failed test for cargo deny (and master too), I'll try to fix it after 1-2 hours.

Reply:
@fanatid clippy did catch this. I saw the check fails and thought it's cargo-deny-action and ignored it.